src/app/home/home-router.module.ts

```diff
@@ -16,6 +16,9 @@ const routes: Routes = [
       { path: 'associated/:planet/:nation', component: NationComponent },
       { path: 'associated/:planet', component: NationComponent },
       { path: 'manager', component: ManagerDashboardComponent },
+      { path: 'manager/meetups', loadChildren: '../meetups/meetups.module#MeetupsModule', data: { parent: true } },

```

Comment:
@paulbert  should we use manager-dashboard module and add routing there ?

Reply:
If possible it would be better to have this in a module, yes.